packages/docs/ssr/nuxt.md

```diff
@@ -47,7 +47,7 @@ By default `@pinia/nuxt` exposes one single auto import: `usePinia()`, which is
 // nuxt.config.js
 export default {
```

Comment:
can you add the defineNuxtConfig as above?

Reply:
sure, will add it